webserver/Migrations/20231003160237_REmodeling.Designer.cs

```diff
@@ -241,6 +241,18 @@ protected override void BuildTargetModel(ModelBuilder modelBuilder)
                     b.Property<decimal>("Price")
                         .HasColumnType("decimal(18,2)");
 
+                    b.Property<int>("area")
+                        .HasColumnType("int");
+
+                    b.Property<int>("numBedrooms")
+                        .HasColumnType("int");
+
+                    b.Property<int>("percentage")
+                        .HasColumnType("int");
```

Comment:
Are you sure this isn't a float?

Reply:
Yes yes. Perhaps i should've documented it better

It goes from 0% to 100%. <100% means the Real Estate is still in construction
I didn't see much point in having it being 'float'. Took the example from another Real Estate Companies website